src/rev_ai/apiclient.py

```diff
@@ -137,6 +139,10 @@ def submit_job_url(
                                                    segments_to_transcribe, speaker_names,
                                                    source_config, notification_config,
                                                    skip_postprocessing,
+						   remove_atmospherics,
```

Comment:
is this correct you added params except `diarization_type`?

Reply:
It appears that they were declared on the prototype but were not actually submitted....

Reply:
@eugenep-rev do you know anything about this?

I just can guess that was intentionally, but looks like a bug